_layouts/packages.html

```diff
@@ -127,12 +127,14 @@ <h2>{{ page.description }}</h2>
 {% endif %}
 </dl>
 
-
+{% if page.downloads.size >= 1 %}
 <h4>Download</h4>
 <p>
-    [<a href="{{ manual }}/README.{{ page.name | downcase }}">README</a>]&nbsp;&nbsp;&nbsp;&nbsp;{{ page.base-archive-name }}
```

Comment:
Can we add back the README link, too? It should also be derivable from the metadata

Reply:
Sure.

Reply:
Actually, I don't think I can do this in the current setup.

The YAML in a `_Packages/<pkgname>.html` file gives the name of the README within the package archive. But the file does not currently include the field `README_URL` of the `PackageInfo.g`. (Moreover, the README at that link might not correspond to the version on the GAP website). Eventually we will be using `package-infos.json`, which will have this information, but we're not yet.

Previously, READMEs were being copied to (e.g.) https://www.gap-system.org/Manuals/pkg/cvec-2.7.4/README.cvec but that's not the case now. So without this I don't think there's a good solution at the moment.

Reply:
OK. I think we really don't need this README link, though. I.e., I am happy to just drop it!